### DISCUSSION TITLE
lottie Implementation

### DISCUSSION THREAD

**Discussion by randydom** [MagicFoundation/Alcinoe#233] [Ideas]
Hello , 1st of all thank you for such a great library , Can you please integrate lottie animation feature ; here we've 2 options :
either by using the official [**lottie**](https://github.com/airbnb/lottie-android) or even better by using the fast cross-platform [**rlottie**](https://github.com/Samsung/rlottie) .
Than you again

**Comment by Spelt** [comment]
Why don't you do it yourself?  Its a complete different project.

**Reply by randydom** [reply]
> Why don't you do it yourself? Its a complete different project.

Do you mean integrate it with  " alcinoe " library or as a new complete project ?
